concepts/strings/links.json

```
@@ -1,2 +1,18 @@
-{
-}
\ No newline at end of file
+[
+  {
+    "description": "string",
+    "url": "https://code.haxe.org/category/beginner/strings.html"
+  },
+  {
+    "description": "string-api",
```

The descriptions are displayed as the link text on the website. You're not limited to snake case or anything. I would suggest making them a bit more easy to read, so maybe:

```suggestion
    "description": "String API",
```